BST.java

```diff
@@ -0,0 +1,59 @@
+/*
+Time Complexity : O(n) where n is time taken to find whether it's BST or not.
```

Comment:
Please check what `n` indicates.

Reply:
Here n means, the amount of time we had taken to find whether it's BST or not. suppose If it's BST then we have to traverse all the elements in the tree.